examples/vmck.nomad

```diff
@@ -0,0 +1,148 @@
+# TODO: remove hardcoded values
+#       add configurationa like in https://github.com/liquidinvestigations/node/blob/master/templates/drone.nomad
+job "vmck" {
+  datacenters = ["dc1"]
+  type = "service"
+
+  group "imghost" {
+    task "nginx" {
+      driver = "docker"
+      config {
+        image = "nginx:mainline"
+        volumes = [
+          "/usr/share/vmck-images:/usr/share/nginx/html", # TODO: add setting for host volume
```

Comment:
Same, move to issue. It can be a single issue like "Make things configurable" with a bullet list of values to be configured.